cli/src/config/templates.toml

```diff
@@ -237,6 +246,17 @@ if(ref.tracking_present(), surround("(", ")", separate(", ",
 'format_snapshot_operation(op)' = 'format_operation(op)'
 'format_root_operation(root)' = 'separate(" ", root.id().short(), label("root", "root()")) ++ "\n"'
 
+
+'format_operation_oneline(op)' = '''
+  concat(
+    separate(" ", op.id().short(), op.user(), format_time_range(op.time())), " ",
+    op.description().first_line(), " ",
+    if(op.tags(), op.tags() ++ "\n"),
```

Comment:
Perhaps `"\n"` should be added no matter if `op.tags()` exists or not? I think all fields can be moved inside `separate(" ", ..)`.
```
separate(" ",
  op.id().short(),
  ...,
  op.tags(),
)
```

Reply:
I think the "\n" was just unnecessary